uninstall_nameof.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+choice="n"
+read -p "Do you really want to remove name identifier tool, write \"y\" for yes and \"n\" for no: " choice
+
+if [[ "$choice" == "y" ]] || [[ "$choice" == "Y" ]];
+then
+path=$(pwd)
+rmpath=$(sudo rm -r $path)
+    if sudo rm -r /usr/share/nameof && sudo rm /usr/local/bin/nameof && $rmpath  ;then
+        echo "successfully uninstalled the \"nameof\" tool , use \"sudo apt autoremove;sudo apt autoclean\" for removing traces and free up memory"
```

Comment:
Is this necessary? We haven't added any packages using the apt tool in the installation.

Reply:
Yes it is to remove the source-code folder, since we don't know in which directory (s)he downloaded the code.

Reply:
I'm talking about the echo command. I'm not sure if its needed, it could be misleading.

Reply:
are you talking about autoremove and autoclean ?
Yes they are not needed, you can modify the prompt.